usp/web_client/requests_client.py

```diff
@@ -72,17 +72,27 @@ class RequestsWebClient(AbstractWebClient):
     __slots__ = [
         '__max_response_data_length',
         '__timeout',
+        '__proxies',
     ]
 
     def __init__(self):
         self.__max_response_data_length = None
         self.__timeout = self.__HTTP_REQUEST_TIMEOUT
+        self.__proxies = {}
 
     def set_timeout(self, timeout: int) -> None:
         """Set HTTP request timeout."""
         # Used mostly for testing
         self.__timeout = timeout
 
+    def set_proxies(self, proxies):
```

Comment:
Hi, sorry for a late reply!

Looks good, but could you:

* add a `typing` annotation to the `proxies` argument (probably something like `Dict[str, str]`);
* describe what's expected out of the argument in the docstring, i.e. add a short explanation of what's in the key and what's in the value in the `proxies` argument? I know it's all detailed in [*requests* manual](https://requests.readthedocs.io/en/master/user/advanced/#proxies) but we'd like to decouple this module from using *requests* specifically.

Reply:
Hi, just did it.
Please check the syntax of the typing annotation (I wasn't aware of these options until your comment...)
I have also enriched the docstring, though whithout explaining all available syntaxes. The easiest would have been to add a reference to the requests module, but as you want it to be decoupled it's up to you.